### ISSUE TITLE
Issues while parsing xml

### ISSUE THREAD

**Issue by ravikumarsahu** [chinmaydas96/Offline_aadhaar_XML_verification#1]
Hello @chinmaydas96 ,

Getting error while parsing xml file.Can you tell me how did you download the xml.I'm using offical website of aadhar.

**Comment by chinmaydas96** [comment]
Yes, I have used the official instruction to download the XML file. And I am feeding directly the zip file into the program.

**Comment by chinmaydas96** [comment]
They recently changed the format of the XML file.
I am working on parsing the latest XML file, they haven't updated the documentation for parsing the new format XML. Will update you soon.**
